src/core/footer/nav-buttons/about.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+
+import { Link } from 'header/link'
+
+export const About = () => <Grid item md={3} lg={3} xl={3}>
+  <Link>Despre proiect</Link>
+</Grid>
```

Comment:
Should we define the behaviour for `sm` and `xs` breakpoints also? It will allow wrapping the options for everything that's smaller than `sm` breakpoint.

![image](https://user-images.githubusercontent.com/39530866/123537685-6a368e00-d739-11eb-979e-20697b95afb2.png)